cloudify_hostpool/rest/backend.py

```diff
@@ -21,25 +22,36 @@
 from cloudify_hostpool import exceptions
 
 
+_DB_FILE_PATH_ENV = 'CFY_HOST_POOL_DB_FILE'
+
+
 class RestBackend(object):
 
     def __init__(self, pool, db_file_name=None):
         if db_file_name is None:
-            # generate a new file on each initialization
-            db_file_name = 'host-pool-{0}.sqlite'.format(uuid.uuid4())
+            db_file_name = os.environ.get(_DB_FILE_PATH_ENV)
```

Comment:
In future we'll change it to load a configuration file. For now this is enough to allow using nondefault database file paths.